term-project/front/integration-testing/example.spec.ts

```diff
@@ -10,35 +10,35 @@ test("if intern tries to register with non-Brown email, will see error message",
   await page.goto("http://localhost:5173/register");
   await page.locator("#root").click();
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your name here")
     .click();
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your name here")
     .fill("Angel Chukwuma");
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your Brown email here")
     .click();
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your Brown email here")
     .fill("angel_chukwuma@gmail.com");
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your password here")
     .click();
   await page
-    .getByLabel("You can registration as a student here")
+    .getByLabel("You can register as a student here")
     .getByPlaceholder("Enter your password here")
     .fill("abcd1234");
   await page.getByPlaceholder("Enter your Work Address here").click();
```

Comment:
We shouldn't need to click this since it's an input, and I think work address should be lowercased.